sources/packages/backend/libs/services/src/sfas/sfas-individual.service.ts

```diff
@@ -41,6 +41,54 @@ export class SFASIndividualService {
     return individual;
   }
 
+  /**
+   * Get SFAS student partial match, if available.
+   * At least two of the three parameters must match.
+   * @param lastName student last name.
+   * @param birthDate student data of birth.
+   * @param sin student Social Insurance Number.
+   * @returns SFAS Student.
+   */
+  async getIndividualStudentPartialMatch(
+    lastName: string,
+    birthDate: string,
+    sin: string,
+  ): Promise<SFASIndividual> {
+    const individual = await this.sfasIndividualRepo
```

Comment:
Minor, not a blocker, usually for these "direct-query-return" cases the preference is just a direct return (`await` would not be needed also).
```
return this.sfasIndividualRepo(
```

Reply:
updated